### DISCUSSION TITLE
Possible for Tuck Function?

### DISCUSSION THREAD

**Discussion by AjayShanker-geek** [rxhanson/Rectangle#498] [Ideas]
It will be great to add a tuck function.
Scenario: Able to tuck apps at the sides of the screen. Personally, I will tuck both my Telegram and WhatsApp app. Toggle them when needing to reply.

Example: 
[Slidepad](https://slidepad.app)
[Tuck](https://www.irradiatedsoftware.com/tuck/)

**Comment by rxhanson** [comment]
Thanks for the suggestion - I've actually been giving this one some thought before you posted this. At the moment, I don't have the bandwidth for it, but I would be inclined to merge a PR with it.
